tests/src/Kernel/ValueObject/ImageValueObjectTest.php

```diff
@@ -56,44 +63,63 @@ protected function setUp(): void {
 
     // Copy file in public files to use it for styling.
     \Drupal::service('file_system')->copy(__DIR__ . '/../../../fixtures/example_1.jpeg', 'public://example_1.jpg');
-  }
 
-  /**
-   * Test the image value object has the correct style applied.
-   */
-  public function testFromStyledImageItem() {
     // Create image file.
     $image = File::create([
       'uri' => 'public://example_1.jpg',
     ]);
     $image->save();
 
-    // Create a test style.
-    /** @var \Drupal\image\ImageStyleInterface $style */
-    $style = ImageStyle::create(['name' => 'main_style']);
-    $style->save();
-
     // Create an image item.
-    $alt = $this->randomString();
-    $title = $this->randomString();
-    $entity = EntityTest::create([
+    $this->entity = EntityTest::create([
       'name' => $this->randomString(),
       'field_image' => [
         'target_id' => $image->id(),
-        'alt' => $alt,
-        'title' => $title,
+        'alt' => 'This is an alternative title',
+        'title' => 'This is a title',
       ],
     ]);
-    $entity->save();
+    $this->entity->save();
+  }
 
-    $object = ImageValueObject::fromStyledImageItem($entity->get('field_image')->first(), $style->getName());
-    $this->assertEquals($title, $object->getName());
-    $this->assertEquals($alt, $object->getAlt());
+  /**
+   * Tests the ::fromStyledImageItem method.
+   */
+  public function testFromStyledImageItem() {
+    // Create a test style.
+    /** @var \Drupal\image\ImageStyleInterface $style */
+    $style = ImageStyle::create(['name' => 'main_style']);
+    $style->save();
+
+    $object = ImageValueObject::fromStyledImageItem($this->entity->get('field_image')->first(), $style->getName());
+    $this->assertEquals('This is a title', $object->getName());
+    $this->assertEquals('This is an alternative title', $object->getAlt());
     $this->assertStringContainsString('/styles/main_style/public/example_1.jpg', $object->getSource());
 
+    // Test that all the cache tags have present and bubbled up.
```

Comment:
```suggestion
    // Test that all the cache tags are present and have bubbled up.
```
same for testFromImageItem()